README.md

```diff
@@ -37,78 +37,81 @@ Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/si
      </dependency>
 ```
 
+**Note** : rdbms:query() and rdbms:cud() function will work only inside a carbon environment
```

Comment:
Lets say WSO2 SP, people will not know "carbon environment"

Reply:
Fixed in ae1b5de